mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -123,9 +123,13 @@ def get_result(self, res):
                         result = self.__find_value(
                             res.pods, 'BasicInformation:PeopleData')
                         if not result:
-                            result = self.__find_value(
-                                res.pods, 'DecimalApproximation')
-                            result = result[:5]
```

Comment:
We should probably trucate the decimals after finding a DecimalApproximation value